### PR TITLE
https://issues.redhat.com/browse/ACM-8583

### PR DESCRIPTION
**Pull request by jc-berger**
See PR for 2.9: https://github.com/stolostron/rhacm-docs/pull/5863/files

### CODE REVIEW

File: clusters/cluster_lifecycle/ansible_config_cluster.adoc [hunk 1]
```diff
@@ -60,6 +60,10 @@ To specify the details about running the Ansible template while creating a templ
 . If you want to initiate any Ansible jobs before the cluster is installed, select *Add an Automation template* in the _Pre-install Automation templates_ section.
 
 . Select between a `Job template` or a `Workflow job template` in the modal that appears. You can also add `job_tags`, `skip_tags`, and workflow types.
++
+* Use the `Extra variables` field to pass data to the `AnsibleJob` resource in the form of `key=value` pairs. 
+
+* Special keys `cluster_deployment` and `machine_pool` are passed automatically as extra variables, containing data on general information about the cluster and details about the worker nodes being created in the cluster.
```
Comment (fxiang1) [inline clusters/cluster_lifecycle/ansible_config_cluster.adoc]: Sorry @jc-berger I made the mistake of mentioning `machine_pool` it should be `install_config` instead.
Please change this line to:

* Special keys `cluster_deployment` and `install_config` are passed automatically as extra variables, containing data on general information about the cluster and details about the cluster install configuration.

**Review by fxiang1 — CHANGES_REQUESTED** [review]

File: clusters/cluster_lifecycle/ansible_config_cluster.adoc (hunk 1 above)
Comment (jc-berger) [reply clusters/cluster_lifecycle/ansible_config_cluster.adoc]: Thanks @fxiang1 ! I applied your feedback. Please let me know if you think it looks good to merge now!

**Review by fxiang1 — APPROVED** [review]
LGTM! Thanks Jake for the quick change!

**Review by cadawson — APPROVED** [review]
Looks good. See suggestions in the 2.9 version. If you made changes there, you might want to make them here, too. Thanks!

**Pull request merged**